stripper/ze_huggy_wuggy_chap4_halloween_rus4.cfg

```diff
@@ -1,3 +1,147 @@
+;stripper by koen (STEAM_1:1:114921174)
+
+;move immunity text to a visible part of the screen
+modify:
+{
+	match:
+	{
+		"classname" "game_text"
+		"targetname" "text_lasers"
+	}
+	replace:
+	{
+		"x" "-1.0"
+		"y" "0.7"
+	}
+}
+
+;rework how immunity text is displayed
+modify:
+{
+	match:
+	{
+		"classname" "filter_activator_name"
+		"targetname" "Filter_None"
+	}
+	delete:
+	{
+		"OnPass" "text_lasersDisplay0-1"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "trigger_hurt"
+		"targetname" "/^mv_2_f_/"
+	}
+	insert:
+	{
+		"OnHurtPlayer" "text_lasersDisplay0-1"
+	}
+}
+
+;remove empty ambient generic in the map
+filter:
+{
+	"classname" "ambient_generic"
+	"origin" "2960 9420 -1976"
+}
+
+;move gravity particle lower to not block vision
+modify:
+{
+	match:
+	{
+		"classname" "info_particle_system"
+		"targetname" "gravity_eff"
+	}
+	replace:
+	{
+		"origin" "-5216 -573.338 1475"
+	}
+}
+
+;fix item particles not showing up when they spawn
+modify:
+{
+	match:
+	{
+		"classname" "info_particle_system"
+		"targetname" "Item_wind_particle_sfera"
+	}
+	replace:
+	{
+		"start_active" "0"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "point_template"
+		"targetname" "Item_wind_template"
+	}
+	insert:
+	{
+		"OnEntitySpawned" "Item_wind_particle_sferaStart3-1"
```

Comment:
Worth verifying, but I seem to recall that `OnEntitySpawned` doesn't fire when the template is spawned through an env_entity_maker. Only when the template is directly spawned. Alternatively you could replace this all by just delaying the entity maker `ForceSpawn`'s in the logic_auto.

Reply:
I tested the stripper locally as well as using it on Zeddy a few hours earlier and it seemed fine, although we did 1-0 the map, meaning I didn't see if it worked on multi-rounds. I'll test it on my test server in a little bit.